demo/src/pages/index.js

```diff
@@ -1,10 +1,98 @@
 import React from 'react';
 
 const Homepage = () => {
+  return (
+    <div style={{ textAlign: 'center', padding: '15px' }}>
```

Comment:
Rather than using inline styles this way, could we use the `emotion` css prop?  Its very similar to what you have here, its just the syntax is a little bit different: [example](https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/components/Layout/Layout.js#L14-L30)

that way, we stay consistent with the components' styling and syntax, happy to answer questions if need be!

Reply:
Yes, thank you. I've switched over to the emotion css prop.